Workflows/GIT/Clean changes.md

```diff
@@ -76,6 +76,29 @@ While on feature branch:
 ```bash
 git fetch origin master
 git rebase -i origin/master
+```
+
+This will start interactive rebase. You can edit it through terminal or your editor of choice.
+To set default editor setup default editor before starting rebase:
+
+```bash
+git config core.editor "{editor-name} --wait"
+```
+
+E.g. for visual studio code:
+
+```bash
+git config core.editor "code --wait"
+```
+
+Refer to [this](https://git-scm.com/book/en/v2/Appendix-C%3A-Git-Commands-Setup-and-Config) guide
+for your editor of choice.
+
+Pick s (squash) for all commits to squash them into a single commit. Edit commit message, save and close editor to finish the rebase.
```

Comment:
I don't really understand this sentence. I can assume what you wanna say because I know what it's about, but beginners will be confused.

Please change the structure of this sentence and I'd suggest to screenshot an interactive rebase with 5 or 6 commits, some flagged as pick and some with `squash` or even `fixup`. The as the last sentence, add this information that it can be automatically done with a flag.